src/site/content/en/fast/codelab-cloudinary-url-api/index.md

```diff
@@ -0,0 +1,334 @@
+      <img src="https://res.cloudinary.com/demo/image/fetch/q_auto,f_auto/https://cloudinary-url-api.glitch.me/images/flower6.jpg" alt="Burgandy bouquet" />
+      <div class="price">Burgandy bouquet - $7</div>
+    </div>
+  </body>
+</html>
+```
+
+## Image Performance Gains
+
+There are impressive web performance gains by using <a href="https://cloudinary.com/documentation/image_optimization">cloud to optimize images</a> without losing quality.
+
+<div class="w-table-wrapper">
+  <table>
+    <caption>
+      Table 1 — The before/after Image sizes comparison.
+    </caption>
+    <tbody>
+      <tr>
+        <th>IMAGE</th>
+        <th>ORIGINAL</th>
+        <th>CLOUDINARY</th>
+        <th>SIZE</th>
+      </tr>
+      <tr>
+        <td>flower1.png</td>
+        <td><span class="w-c--warning">289 KB</span></td>
+        <td><span class="w-c--success">22 KB</span></td>
+        <td><span class="w-c--success">↓ -92.39%</span></td>
+      </tr>
+      <tr>
+        <td>flower2.jpg</td>
+        <td><span class="w-c--warning">59 KB</span></td>
+        <td><span class="w-c--success">19 KB</span></td>
+        <td><span class="w-c--success">↓ -67.8%</span></td>
+      </tr>
+      <tr>
+        <td>flower3.png</td>
+        <td><span class="w-c--warning">367 KB</span></td>
+        <td><span class="w-c--success">38 KB</span></td>
+        <td><span class="w-c--success">↓ -89.65%</span></td>
+      </tr>
+      <tr>
+        <td>flower4.jpg</td>
+        <td><span class="w-c--warning">125 KB</span></td>
+        <td><span class="w-c--success">73 KB</span></td>
+        <td><span class="w-c--success">↓ -41.6%</span></td>
+      </tr>
+      <tr>
+        <td>flower5.jpg</td>
+        <td><span class="w-c--warning">96 KB</span></td>
+        <td><span class="w-c--success">47 KB</span></td>
+        <td><span class="w-c--success">↓ -51.04%</span></td>
+      </tr>
+      <tr>
+        <td>flower6.jpg</td>
+        <td><span class="w-c--warning">68 KB</span></td>
+        <td><span class="w-c--success">24 KB</span></td>
+        <td><span class="w-c--success">↓ -64.71%</span></td>
+      </tr>
+    </tbody>
+  </table>
+</div>
+
+Hooray! These results are much better.
+
+## Lighthouse Audit
+
+Lastly, it's a good idea to use <a target="_blank" rel="noopener noreferrer" href="https://web.dev/discover-performance-opportunities-with-lighthouse">Lighthouse</a> to verify the changes that you just
+made.
+
+Lighthouse's "Efficiently encode images" performance audit can let you know if
+the JPEG images on your page are optimally compressed.
+
+- To preview the site press the **View App** button (in the right-bottom corner).
+- Run the Lighthouse performance audit (Lighthouse ❯ Options ❯ Performance) on
+  the <a target="_blank" rel="noopener noreferrer" href="https://codelab-cloudinary.glitch.me/">live version</a> of your Glitch and verify that the "Efficiently encode
+  images" as well as "Serve images in next-gen formats" audits were passed.
+
+### RESULTS
```

Comment:
I think the "Results" section can be omitted.

Reply:
Should I delete this entire section?

![img](https://on.ahmda.ws/2104a7/c)

Reply:
After re-reading this a bunch of times I've changed my mind. I think this would make more sense to me:

- Keep the "Results" section - but show the optimized Lighthouse result first and the unoptimized result second.
- Remove "optmized.html" so people aren't tempted to skip through the code lab just to get to the flashy Lighthouse score :)

If I were reading this I think I'd run the Lighthouse audit when I got to the last bullet point of "Lighthouse Audit". After that I think I'd expect to see the results of a successful/optimized Lighthouse run (so I could verify that I got the same results and did everything correctly). Then, if that was followed by an image of the Lighthouse report for the unoptimized "before" version, I could see the impact of what I just did, without necessarily having to run Lighthouse again.

Reply:
@khempenius — That makes perfect sense. I've made the following changes:

- Removed `optimized.html` — good call.
- Moved the optimized results to the top.
- Removed the redundant info from the non-optimized results.